config.proto

```diff
@@ -52,7 +52,8 @@ message Reporting {
     // opa describes the setting related to the Open Policy Agent
     Opa opa = 4;
 
-    // reporter type. Defaults to zipkin, in the future it will change to otlp.
+    // reporter type for traces.
+    // Agents should set a default type as ZIPKIN or OTLP while creating default config
```

Comment:
We should agree what the default is. Shall we set it to OTLP?

Reply:
Yes, I am updating the default exporter to otlp in java and go agents. We need OTLP exporter for adding resource attributes in spans. ZIPKIN exporter drops all resource attributes except `service-name`

Reply:
with the comment below, I think you can remove the second line.

Reply:
This is a breaking change, Do we need to introduce this now?